usaspending_api/accounts/views/federal.py

```diff
@@ -0,0 +1,34 @@
+from usaspending_api.accounts.serializers import FederalAccountSerializer
```

Comment:
Thanks for splitting out federal accounts and TAS! I have the same question about calling this file `federal_account.py` instead of `federal.py`